detection_rules/attack.py

```diff
@@ -28,7 +28,7 @@
 def get_attack_file_path() -> str:
     pattern = 'attack-v*.json.gz'
     attack_file = get_etc_glob_path(pattern)
-    if len(attack_file) != 1:
+    if len(attack_file) < 1:
```

Comment:
I'd rather change the error message than the count check - this is also to ensure an additional file is not added

Reply:
Makes since. I'll update the error